modules/fundamental/src/product/endpoints/test.rs

```diff
@@ -30,6 +31,7 @@ async fn all_products(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
             "AMQ Broker",
             ProductInformation {
                 vendor: Some("Red Hat".to_string()),
+                cpe: None,
```

Comment:
maybe worthwhile to populate ?

Reply:
Actually cpe_key is not propagated to the endpoint at the moment. It used only internally for correlation. Maybe we should wait with introducing it into the api until we finish with todo mentioned below?

Reply:
It's being tested in 'service' btw